pkgs/development/libraries/cln/default.nix

```
@@ -1,12 +1,13 @@
-{ lib, stdenv, fetchurl, gmp }:
+{ lib, stdenv, fetchgit, gmp }:
 
 stdenv.mkDerivation rec {
   pname = "cln";
   version = "1.3.6";
 
-  src = fetchurl {
-    url = "${meta.homepage}${pname}-${version}.tar.bz2";
-    sha256 = "0jlq9l4hphk7qqlgqj9ihjp4m3rwjbhk6q4v00lsbgbri07574pl";
+  src = fetchgit {
+    url = "git://www.ginac.de/cln.git";
```

```suggestion
    url = "https://www.ginac.de/cln.git";
```
`git://` is considered unsafe.